stdlib/public/Concurrency/Actor.cpp

```diff
@@ -453,22 +455,17 @@ struct RunningJobInfo {
     Inline, Other
   };
   KindType Kind;
-  JobPriority Priority;
 
   bool wasInlineJob() const {
     return Kind == Inline;
   }
 
-  static RunningJobInfo forOther(JobPriority priority) {
```

Comment:
Since JobPriority of the processing job does not necessarily correlate to the priority of the actor we're processing, or the priority of the thread running that job (as a result of priority escalation that can happen asynchronously or as a result of self-overrides), I decided to remove it from this structured, since we can't make scheduling decisions based on this information.